kie-decision-playground/src/main/assembly/assembly-zip-kie-decision-playground.xml

```diff
@@ -0,0 +1,60 @@
+<!--
+  ~ Copyright 2017 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+  <id>project</id>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <fileSets>
+
+    <fileSet>
+      <useDefaultExcludes>false</useDefaultExcludes>
+      <includes>
+        <include>curriculumcourse/pom.xml</include>
+        <include>dinnerparty/pom.xml</include>
+        <include>employee-rostering/pom.xml</include>
+        <include>mortgages/pom.xml</include>
+        <include>optacloud/pom.xml</include>
+      </includes>
+      <excludes>
+        <exclude>kie-*/pom.xml</exclude>
+      </excludes>
+      <directory>target/kie-decision-playground-xslt</directory>
+      <outputDirectory>/</outputDirectory>
+      <filtered>true</filtered>
+    </fileSet>
+    <fileSet>
+      <directory>../</directory>
+      <useDefaultExcludes>false</useDefaultExcludes>
+      <includes>
+        <include>curriculumcourse/**</include>
+        <include>dinnerparty/**</include>
+        <include>employee-rostering/**</include>
+        <include>mortgages/**</include>
+        <include>optacloud/**</include>
+      </includes>
+      <excludes>
+        <exclude>**/*.iml</exclude>
+        <exclude>*/.gitignore</exclude>
+        <exclude>*/target/**</exclude>
+        <exclude>*/pom.xml</exclude>
```

Comment:
Typically a non-root .gitignore looks like this:

```
/target
/local

# Eclipse, Netbeans and IntelliJ files
/.*
!.gitignore
/nbproject
/*.ipr
/*.iws
/*.iml

```

This list is different on a few things:
- no nbprojects or .* exclude. Some people might build with Netbeans or Eclipse.
- The gitignore file is being excluded, but in normally it is not excluded. Doubt it matters much though.
- local dir: by convention, a directory that is not cleaned with "mvn clean" but is not commited into git. Not everyone uses it, but several of us do.